optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -717,10 +855,16 @@ def __init__(self, config: LlamaConfig):
         self.padding_idx = config.pad_token_id
         self.vocab_size = config.vocab_size
 
+        self.distributed_strategy = config.distributed_strategy
```

Comment:
You need to add `distributed_strategy` to `LlamaConfig` here: https://github.com/huggingface/optimum-habana/blob/main/optimum/habana/transformers/models/llama/configuration_llama.py
You can follow what is done for `mlp_bias` and `fused_qkv`.

Reply:
Done

Reply:
Hmm I don't see this change

Reply:
I had removed this change  self.distributed_strategy = config.distributed_strategy
so it thought it was not necessary since i dint see it getting used any where.
I have updated the  LlamaConfig now.

Reply:
Yeah, it may not be absolutely necessary but I feel like it's cleaner that way.